Automattic/wp-calypso
# Support site_tagline in design previews

#### Proposed Changes

This PR updates the site preview so that `site_tagline` is passed when:

1. The site intent is either Write or Sell.
2. Users have set a site tagline during one of the previous onboarding steps. See screenshots below for reference.

Write flow:
![Screen Shot 2022-07-28 at 3 59 39 PM](https://user-images.githubusercontent.com/797888/181675294-01ad1eda-88f5-406f-bcc1-b8025ef43aaf.png)

Sell flow:
![Screen Shot 2022-07-28 at 3 59 54 PM](https://user-images.githubusercontent.com/797888/181675299-53e479bc-1e0b-42b6-955f-c8de6c4623f2.png)

Note that this PR depends on D84981-code.
Also, see D84981-code for the list of themes that support customizing the site tagline.

#### Testing Instructions



* Head to `/setup?siteSlug=${site_slug}`
* Either select "Write & Publish" (Write flow) or "Sell online" (Sell flow) and click on the "Continue" button.
* Continue the onboarding flow and type in a site tagline when prompted.
* Continue the onboarding flow until the design picker screen.
* Click on any of theme that supports site tagline to preview it.
* Ensure that the site tagline is replaced by the one you provided.

#### Pre-merge Checklist






Related [#65515](https://github.com/Automattic/wp-calypso/issues/65515)
